members/members.json

```diff
@@ -6,6 +6,7 @@
     "website": "https://www.dlsc.com",
     "linkedIn": "dlemmermann",
     "twitter": "dlemmermann",
+    "bluesky": "https://bsky.app/profile/dlemmermann.bsky.social",
```

Comment:
Maybe we could limit ourselves to the username only ?

```
"bluesky" = dlemmermann.bsky.social
```

Reply:
As Bluesky is distributed, don't we need the full address, similar to Mastodon?

Reply:
https://techcrunch.com/2023/07/05/bluesky-announces-its-8m-seed-round-first-paid-service-custom-domains/
"ability for users to set a custom domain as their handle"

Not sure if this is about the first or last part of the handle ;-)

Reply:
This is for the last part

Example :
https://bsky.app/profile/stephanjanssen.be

![image](https://github.com/user-attachments/assets/a8df6c2d-ba67-4e0f-a1a6-08a7da644c8e)

Reply:
Thanks for the info! Code and data are updated